cli/command/utils.go

```diff
@@ -123,6 +122,8 @@ func PromptForConfirmation(ctx context.Context, ins io.Reader, outs io.Writer, m
 
 	select {
 	case <-notifyCtx.Done():
+		// print a newline on termination
+		fmt.Fprintln(outs, "")
```

Comment:
As this is new code, might as well please the linters;

```suggestion
		// print a newline on termination
		_, _ = fmt.Fprintln(outs, "")
```